src/GuessControl.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component, useState } from "react";
+import React from "react";
```

Comment:
did you push your changes to main? I can't see all the newest changes you made on this file based on this PR.

Reply:
When I started this assignment I forgot to make a branch. And made one pretty late.  I did end up making several changes on the branch so I'm not sure why it's saying that. I'll take a look to make sure I pushed all my changes.